test_files/enum/enum.ts

```diff
@@ -61,7 +61,13 @@ enum StringEnum {
   OTHER_STR = 'xyz',
 }
 
+enum StringKeyEnum {
```

Comment:
can you add comments to the enums explaining exactly what case this is testing?

Reply:
👍